src/RestServer/Exceptions/RestErrorCodes.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2015-2023 The Neo Project.
+//
+// The Neo.Plugins.RestServer is free software distributed under the MIT software license,
+// see the accompanying file LICENSE in the main directory of the
+// project or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+namespace Neo.Plugins.RestServer.Exceptions
+{
+    internal static class RestErrorCodes
+    {
+        //=========================Rest Codes=========================
```

Comment:
The API is different, but https://github.com/neo-project/proposals/pull/156 is likely still applicable, just two codes are not really usable.

Reply:
Right in the proposal. We should have some unification between REST and RPC codes.